src/main/java/uk/gov/hmcts/befta/dse/ccd/definition/converter/JsonTransformer.java

```diff
@@ -35,6 +36,8 @@
             "WorkBasketInputFields", "WorkBasketResultFields", "Category", "Banner", "CaseType", "ComplexTypes", "EventToComplexTypes",
             "FixedLists", "Jurisdiction", "UserProfile","SearchAlias", "SearchCasesResultFields");
 
+    private static final List<String> SHEETS_FOR_URL_SUBSTITUTIONS = Arrays.asList("BEFTA_MASTER/FT_MasterCaseType/CaseEvent.json");
+
```

Comment:
better to allow substitution in the same spreadsheets of different files. not only in befta master file.

Reply:
Fixed in https://github.com/hmcts/befta-fw/pull/93/commits/d3e7ac8024c06c6cb06ba875509748598cae534b